### ISSUE TITLE
Barchart is not visible

### ISSUE THREAD

**Issue by hassan31** [nuwehq/nuwe-charts-ios#5]
Hey guys, I'm facing an issue while running the application, unfortunately the bar chart is not working, even I can't see the bar chart either I rotate my device. Also, I tried to set its delegate and implemented the methods that returns values to show a bar chart... Can you please guide me on this, how to show the bar chart. May be I'm trying a wrong way...

Thanks in advance.

**Comment by BBHNation** [comment]
Yes I have the same issue.I just follow the wiki to build a bar chart but it doesn't show.I think it may mot be the point of us,it's the problem of this framework.

**Comment by Black-Colt** [comment]
In the example he didn't init the bar chart.
I fixed it by init it with its own frame, If you need I can upload example.

**Comment by AnilkumarRabhasa** [comment]
@Black-Colt I am trying to load this barchart in my tableview cell, but its not loading even barchart delegate methods are calling, can you help me on this, if you have awareness on this? Please check this for more info, I already raised ticket on this following.
https://github.com/nuwehq/nuwe-charts-ios/issues/6
